api_tests/Delete_authorization.yaml

```diff
@@ -0,0 +1,50 @@
+# Test generated by RoostGPT for test credwallet-integration-artillery using AI Type Open AI and AI Model gpt-4
+
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  payload:
+    path: "Delete_authorization.csv"
+    fields:
+      - "id"
```

Comment:
fields are expanded.

Reply:
Phase started: unnamed (index: 0, duration: 1s) 12:57:54(+0530)

Phase completed: unnamed (index: 0, duration: 1s) 12:57:55(+0530)

Given authorization ID
* POST /authorizations/1 

  not ok statusCode 405 
  expected: 200
       got: 405
  not ok hasProperty response body has no id property 
  expected: id
       got: response body has no id property
  Request params:
    http://localhost:4012/authorizations/1
    ""
  Headers:
  access-control-allow-origin: *
  access-control-allow-headers: *
  access-control-allow-credentials: true
  access-control-expose-headers: *
  content-type: application/problem+json
  content-length: 231
  date: Wed, 20 Dec 2023 07:27:55 GMT
  connection: keep-alive
  keep-alive: timeout=5
  Body:
{"type":"https://stoplight.io/prism/errors#NO_METHOD_MATCHED_ERROR","title":"Route resolved, but no method matched","status":405,"detail":"The route /authorizations/1 has been matched, but it does not have \"post\" method defined"}
  User variables:
    target: http://localhost:4012
    $environment: undefined
    id: 1
    $uuid: 48ab974f-7403-40a7-a2f5-9c466cf85fd8
--------------------------------------
Metrics for period to: 12:58:00(+0530) (width: 0.035s)


wrong method